mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -36,6 +36,10 @@ func resourceStorageBucket() *schema.Resource {
 			customdiff.ForceNewIfChange("retention_policy.0.is_locked", isPolicyLocked),
 		),
 
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(4 * time.Minute),
```

Comment:
I didn't want to extend the scope of the PR mainly focusing on fixing [this issue](https://github.com/hashicorp/terraform-provider-google/issues/10423), but happy to add other timeouts (Create/Update/Delete) here as well.

Reply:
Hi @turkenh thanks for working on this. If you wouldn't mind, adding the other timeouts should be helpful as well.

Have you confirmed that adding this removes the issue?

Reply:
@c2thorn added all timeouts.

> Have you confirmed that adding this removes the issue?

I've tested and confirmed that with the default timeouts here, it takes 4m (instead of 20m) with the case described in the issue.
Now timeouts being configurable, I've also verified that providing even a smaller timeout, e.g. 1m, works as expected.